frontend/src/components/common/PageTitle.vue

```diff
@@ -11,11 +16,23 @@ const props = defineProps({
     subtitle: {
         type: String,
         required: true,
-    },
+    }
 });
 
 const route = useRoute();
 
+const isActive = (item: MenuItem) => {
```

Comment:
we don't need to make an extra method for this

Reply:
you're correct, it's not needed anymore. Thanks!